src/state-mutation.js

```diff
@@ -59,6 +60,16 @@ export const makeUpdateReverseRelationship = (
       )
     ) {
       return foreignResources;
+    } else if (Array.isArray(foreignResourceRel) && !newRelation) {
+      const relIdx = foreignResourceRel.findIndex(item => (
+        item.id === resource.id
+      ));
+
+      if (foreignResourceRel[relIdx]) {
+        foreignResourceRel.splice(relIdx, 1);
```

Comment:
I'm not sure it's a good idea to mutate `foreignResourceRel`, as it may lead to unexpected behavior. Should be possible to just return from here?